src/brr.ml

```diff
@@ -1645,6 +1645,9 @@ module Window = struct
 
   (* Operations *)
 
+  let open' ?(name = Jstr.empty) ?(features = Jstr.empty) w u =
+    Jv.to_option Fun.id @@ Jv.call w "open" [|Uri.to_jv u; Jv.of_jstr name; Jv.of_jstr features|]
```

Comment:
That doesn't seem to fit 80 columns. It should.

Reply:
Updated.